src/pages/api/v1/job-listings/index.ts

```diff
@@ -0,0 +1,55 @@
+import prisma from '../../../../lib/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const getJobListings = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const data = await prisma.jobListing.findMany();
+    if (!data) {
```

Comment:
I think we might have to test this at a later date,

I wonder if Prisma returns an empty array or an error.

Let's not make any changes for this one, but I wanted to point that out.

Reply:
![image](https://user-images.githubusercontent.com/6619450/212429955-5d34047a-30ae-4bf7-bf9f-88e236922832.png)
It will returns an error.

Reply:
Accidentally I deleted the branch, but lucky Git allowed me to restore it and re-open the pull request.

Reply:
Well done with figuring out how to get the deleted branch back. :+1: